packages/back-end/src/tracing.ts

```diff
@@ -36,6 +39,11 @@ const sdk = new opentelemetry.NodeSDK({
   instrumentations: getNodeAutoInstrumentations(),
   resourceDetectors: getResourceDetectors(),
   metricReader,
+  resource: new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: "growthbook",
+    [SemanticResourceAttributes.SERVICE_NAMESPACE]: "backend",
+    [SemanticResourceAttributes.SERVICE_VERSION]: getBuild().sha,
+  }),
```

Comment:
These are good default values. If env vars are specified for any of these, they should take precedence. Specifically service name. That may already be happening (I didn't test it), but just noting it here.